frontend/app/updateRollButton.test.js

```diff
@@ -33,7 +33,32 @@ describe('Disable roll button ', () => {
     const mockFileResponse = {
         status: 200,
         readyState: 4,
-        responseText: 'true',
+        responseText: '{"player_id": [2, 1]}',
+    };
+
+    beforeAll(() => {
+        document.body.innerHTML = '<button type="button" id="roll">Roll die</button>';
+    });
+
+    afterAll(() => {
+        document.body.innerHTML = oldDocumentBody;
+    });
+
+    test(' disable button ', (done) => {
+        functionCall.checkRollValuesEqual(mockFileResponse);
+        expect(document.body.innerHTML).toEqual('<button type="button" id="roll" disabled="">Roll die</button>');
+        done();
+    });
+});
+
+describe('Enable roll button ', () => {
+    const oldDocumentBody = document.body.innerHTML;
+
+    // simulate response
+    const mockFileResponse = {
+        status: 200,
+        readyState: 4,
+        responseText: '{"player_id": [2, 2]}',
```

Comment:
This should also be changed
```javascript
responseText: '{"your_rolls": [2, 2]}',
```

Reply:
I think originally `"player_id"` was actually meant to be the value of the player_id, rather than a string.

Nevertheless, if it's going to stay as a string, then yes, it's worth making it `"your_rolls"`